spec/rspec/core/did_you_mean_spec.rb

```diff
@@ -0,0 +1,42 @@
+module RSpec
+  module Core
+    RSpec.describe DidYouMean do
+      describe '#call' do
+        if defined?(::DidYouMean::SpellChecker)
```

Comment:
You should be able to remove this

Reply:
You need to remove this so the tests run when not defined for not defined

Reply:
done